### ISSUE TITLE
Cannot disable header output

### ISSUE THREAD

**Issue by Sengrith** [CircleOfNice/CiRestClientBundle#45]
when CURLOPT_HEADER is is set to true, the response still give back header in output.

**Comment by Timo-Linde** [comment]
Yes we are overwriting this option because we need the header for the response object.

https://github.com/CircleOfNice/CiRestClientBundle/blob/master/Services/Curl.php#L89

**Comment by TobiasHauck** [comment]
As Timo says this is the only option we really need to set by default. What exactly do you wanna do? Maybe we have missed something ;)
